IndicatorFields/incidentfield-firstseenbyfeed.json

```diff
@@ -0,0 +1,45 @@
+{
+    "id": "indicator_firstseenbyfeed",
+    "version": -1,
+    "modified": "2020-02-06T16:34:43.859782506Z",
+    "sortValues": null,
+    "vcShouldIgnore": false,
+    "commitMessage": "",
+    "shouldCommit": false,
+    "name": "First Seen By Feed",
+    "prevName": "First Seen By Feed",
```

Comment:
Up to you if you think it makes sense to change these to `First Seen By Feed Vendor`. Same goes for the `Last Seen` one.

Reply:
I think its currently makes sense as is